### ISSUE TITLE
Prompt user to upgrade `shop=yes` and other generic presets

### ISSUE THREAD

**Issue by matkoniecz** [openstreetmap/iD#5955]
shop=yes can be always replaced by a better tag and unfortunately it is quite popular - currently it is 7th most popular value, with 149k uses https://taginfo.openstreetmap.org/keys/shop#values !

EDIT: equivalent issue in JOSM - see https://josm.openstreetmap.de/ticket/17377#ticket

**Comment by bhousel** [comment]
I agree, if `shop=yes` is the only primary tag.. 

Weird that it’s so common!  I wonder if this is a tag that people like to stick on other features instead of mapping the actual thing (`tourism=museum`+`shop=yes`?)

**Comment by matkoniecz** [comment]
Right, about 6.35%  is combined with `amenity=fuel` (and in this case it is not a blatant problem, though it would be probably better to tag shop as a separate object) https://taginfo.openstreetmap.org/tags/shop=yes#combinations

But other 139k uses has it as a main tag.

**Comment by HolgerJeromin** [comment]
amenity=fuel + shop=convenience/kiosh is in most cases better. Even on the same object

**Comment by bhousel** [comment]
> amenity=fuel + shop=convenience/kiosh is in most cases better. Even on the same object

In this situation we are encouraging mappers to create separate features for the fuel station (grounds) and the shop (poi).

**Comment by matkoniecz** [comment]
shop=yes as a primary object may be also worth using special icon indicating problem (in red? with an exclamation mark? question mark?).

**Comment by matkoniecz** [comment]
https://josm.openstreetmap.de/ticket/17377#comment:4 - JOSM now warns about `shop=yes`, except where it is with `amenity=fuel` (in trunk, this version is not yet released)

**Comment by matkoniecz** [comment]
I think that after all shop=yes getting "(Unspecified Type)" should be enough

maybe should get also icon with question mark?

**Comment by matkoniecz** [comment]
but that would be for id tagging schema
